client/src/main/java/com/networknt/client/ssl/CientX509TrustManager.java

```diff
@@ -0,0 +1,62 @@
+package com.networknt.client.ssl;
+
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Objects;
+import java.util.Set;
+
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * Customized X509TrustManager. 
+ * Note: X509TrustManagers are eventually converted to X509ExtendedTrustManagers by sun.security.ssl.SSLContextImpl.AbstractTrustManagerWrapper
+ * 
+ * @author Daniel Zhao
+ *
+ */
+public class CientX509TrustManager implements X509TrustManager{
```

Comment:
I think some method level comments explaining what each of them do would go a long way

Reply:
Method specifications are available in the javadoc of X509TrustManager. I'll add references to the interface.